sunpy/map/mapbase.py

```diff
@@ -2129,11 +2129,14 @@ def draw_quadrangle(self, bottom_left, *, width: u.deg = None, height: u.deg = N
 
         Parameters
         ----------
-        bottom_left : `~astropy.coordinates.SkyCoord`
-            The bottom-left coordinate of the quadrangle. It can
-            have shape ``(2,)`` to simultaneously define ``top_right``.
-        top_right : `~astropy.coordinates.SkyCoord`
-            The top-right coordinate of the quadrangle.
+        bottom_left : `astropy.units.Quantity` or `~astropy.coordinates.SkyCoord`
+            The bottom-left coordinate of the rectangle. If a `~astropy.coordinates.SkyCoord` it can
+            have shape ``(2,)`` and simultaneously define ``top_right``. If specifying
+            pixel coordinates it must be given as an `~astropy.units.Quantity`
+            object with units of `~astropy.units.si.pix`.
```

Comment:
```suggestion
            object with pixel units (e.g., ``pix``).
```
There's no good intersphinx link